### PR TITLE
Update Usage by hours of the day.workbook

### LINKED ISSUE #1930: Usage through the day Workbook invalid order by
The _Usage through the day_ Workbook orders by the string variable used for display purposes.

KQL Snippet:
| extend Day = strcat(monthofyear(timestamp), '/', dayofmonth(timestamp), ' - ', case(WeekDay == 0d, 'Sun', WeekDay == 1d, 'Mon', WeekDay == 2d, 'Tue', WeekDay == 3d, 'Wed', WeekDay == 4d, 'Thu', WeekDay == 5d, 'Fri', 'Sat'))
| extend Day1 = bin(timestamp, 1d)
| order by **Day** asc, Hour asc

Visualization screenshot:
image

Potential solution KQL snippet:
| extend Day = strcat(monthofyear(localTimestamp), '/', dayofmonth(localTimestamp), ' - ', case(WeekDay == 0d, 'Sun', WeekDay == 1d, 'Mon', WeekDay == 2d, 'Tue', WeekDay == 3d, 'Wed', WeekDay == 4d, 'Thu', WeekDay == 5d, 'Fri', 'Sat'))
| extend Day1 = bin(localTimestamp, 1d)
| order by **localTimestamp** asc, Hour asc

And the visualization:
image

Happy to submit a PR for this.

### PR DESCRIPTION
**Pull request by jackwagons**
Switched order by from display string to date.  Screenshots provided in  #1930 
Fixes #1930 

## PR Checklist

* [X] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Usage/Usage by hours of the day/Usage by hours of the day.workbook [hunk 1]
```diff
@@ -100,7 +100,7 @@
...
+        "query": "let start = startofday({TimeRange:start});\nunion customEvents, pageViews\n| where timestamp >= start\n| where name in ({Activities}) or ('*' in ({Activities})) or ('#' in ({Activities}) and itemType == 'pageView') or ('&' in ({Activities}) and itemType == 'customEvent')\n//| summarize {Metric} by bin(timestamp, 2h)\n| make-series {Metric} default = 0 on timestamp from startofday({TimeRange:start}) to endofday({TimeRange:end}) step 2h\n| mv-expand Metric to typeof(int), timestamp to typeof(datetime)\n| extend Hour = toreal(hourofday(timestamp)) / 100, WeekDay = dayofweek(timestamp)\n| where '{ExcludeWeekends}' == 'No' or (WeekDay != 0d and WeekDay != 6d)\n| extend Day = strcat(monthofyear(timestamp), '/', dayofmonth(timestamp), ' - ', case(WeekDay == 0d, 'Sun', WeekDay == 1d, 'Mon', WeekDay == 2d, 'Tue', WeekDay == 3d, 'Wed', WeekDay == 4d, 'Thu', WeekDay == 5d, 'Fri', 'Sat'))\n| extend Day1 = bin(timestamp, 1d)\n| order by localTimestamp asc, Hour asc\n| project-away timestamp, WeekDay, Day1\n| serialize\n| evaluate pivot(Hour, sum(Metric))\n//| order by Day1 asc\n//| project-away Day1",
```
Comment (MSKaiLian) [inline Workbooks/Usage/Usage by hours of the day/Usage by hours of the day.workbook:103]: what is this localTimestamp,  you mean "timestamp"?

Comment (jackwagons) [reply Workbooks/Usage/Usage by hours of the day/Usage by hours of the day.workbook]: Ok yeah my apologies, fixed,

Comment (jackwagons) [reply Workbooks/Usage/Usage by hours of the day/Usage by hours of the day.workbook]: @MSKaiLian, btw that was from a clone where I'm hard coding a time diff to get local time.  Is there an existing implementation of adding the _Local_ time to Workbooks?

Example from Transaction Search:
image

Comment (MSKaiLian) [reply Workbooks/Usage/Usage by hours of the day/Usage by hours of the day.workbook]: Seem workbook time picker doesn't have the feature.

Comment (jackwagons) [reply Workbooks/Usage/Usage by hours of the day/Usage by hours of the day.workbook]: Ok, thanks

Comment (jackwagons) [reply Workbooks/Usage/Usage by hours of the day/Usage by hours of the day.workbook]: Are you good with resolving this conversation?

**Comment by jackwagons** [comment]
@MSKaiLian, is there anything needed from my side at this point?

**Review by MSKaiLian — APPROVED** [review]

**Pull request merged**